vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -148,7 +148,7 @@ std::vector<facebook::react::NativeModuleDescription> GetModules(
 
   modules.emplace_back(
     "UIManager",
-    [uiManager = std::move(uiManager)]() { return facebook::react::createUIManagerModule(uiManager); },
+    [uiManager = std::move(uiManager)]() { return facebook::react::createBatchingUIManagerModule(uiManager); },
     messageQueue);
```

Comment:
I wonder if having a different queue here is better than if we have a batching queue and then in the uwp nativeuimanager onbatchcomplete we can drain it..?  with both approaches it would be be nice if we could start running ui thread actions as soon as the first thing comes in, in case it takes a long time for the batch complete to arrive... if we wanted to optimize for that and potentially keep the ui thread locked until we see the batch end

Reply:
I talked to Keith some about this and the part about starting actions before we have received the end batch is probably not critical,  but having a different queue implementation would hopefully simplify this so we don't need to wrap everything

Reply:
Per our discussion, I think we can look at the different queue implementation after this goes in. One reason this is focused on the UIManager was to limit its impact on any other calls that come through that queue.